Trident/offsetfinder.c

```diff
@@ -832,11 +858,13 @@ uint32_t find_task_for_pid(void) {
         case iPhone41_iOS932: return 0x2fcd80;
         case iPhone41_iOS933: return 0x2fcec0;
         case iPhone41_iOS934: return 0x2fcec0;
+        case iPhone51_iOS933: return 0x302fd4;
         case iPhone52_iOS920: return 0x2fbb8c;
         case iPhone52_iOS921: return 0x2fbc9c;
         case iPhone52_iOS930: return 0x302e3c;
         case iPhone52_iOS932: return 0x302df0;
         case iPhone52_iOS933: return 0x302fd4;
+        case iPhone52_iOS934: return 0x302fd4;// ? to be verified
```

Comment:
There should be a check with zero (pid == 0) at the beginning, followed by a call to proc_find, can you confirm?